core/templates/dev/head/pages/exploration_editor/EditorServicesSpec.js

```diff
@@ -263,7 +263,8 @@ describe('Change list service', function() {
             }
           },
           visible_in_states: ['newState1']
-        }
+        },
+        panel: undefined
```

Comment:
It is strange isn't it.? But the test was returning this panel: undefined attribute that the mock was not handling. I'm not sure why this showed up now?